.github/workflows/ci.yaml

```diff
@@ -17,15 +17,63 @@ on:
         default: ""
 
 jobs:
+  skipper:
+    runs-on: ubuntu-latest
+    outputs:
+      skip_build: ${{ steps.skipper.outputs.skip_build }}
+      skip_test: ${{ steps.skipper.outputs.skip_test }}
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ inputs.commit-hash }}
+          fetch-depth: 0
+      - id: skipper
+        env:
+          TARGET_BRANCH: ${{ inputs.target-branch }}
+        run: |
```

Comment:
I would extract this to its own script file.

Reply:
Definitely, I only noticed it's used multiple times after sending the PR.

Reply:
Not only the `run` part. The full description of the `skipper` job is duplicated as well.